chrome-ext/apiHandler.js

```diff
@@ -0,0 +1,15 @@
+class APIHandler {
+    constructor() {
+        this.PICTURE_BASE_URL = "https://www.metaweather.com";
+        this.backgroundApp = axios.create({ baseURL: this.PICTURE_BASE_URL })
+
+    }
+
+    //did not work due to
+    getWeather() {
+        this.backgroundApp.get("/api/location/766273/")
+            .then(response => console.log(response.data))
+        return null
+    }
```

Comment:
Ok, a few thoughts here:
- I was worried about running into a CORS issue myself (i threw out a few ideas because of this)
- I've been able to get around these before using a proxy server (ie: a request to a server that makes a request to the api). I'm not sure how this would be done in a chrome extension (I ran into some other permission issues as well), but I think it could work

Good attempt. Glad to see you 'go big', even if it didn't quite work out!

Reply:
yeah... i once use the https://cors-anywhere.heroku prefix. i don't know if you know it, it helped me once to go around a google places core issue..